pkg/api/v1/types.go

```diff
@@ -1152,6 +1152,8 @@ type ServiceSpec struct {
 	// users to handle external traffic that arrives at a node.
 	DeprecatedPublicIPs []string `json:"deprecatedPublicIPs,omitempty" description:"deprecated. externally visible IPs (e.g. load balancers) that should be proxied to this service"`
 
+	LoadBalancerIP string `json:"loadBalancerIP, omitEmpty`
```

Comment:
I was very confused until I worked out how this differed from the ExternalIPs :/
No, OpenStack LBaaS doesn't expose a way to specify chassis separate from external IPs.

1. I strongly suspect this should be an opaque provider-specific string rather than implying that its an IP.

2. I feel like this is very much a provider-specific implementation detail, and should go in some provider-specific config file or some other generic key/value provider-attributes-extension mechanism.

Eg: the k8s openstack config currently requires a "subnet-id" and other monitoring options for loadbalancers.  These options are set globally via the config right now, but there are reasons you might want to vary them by service.